dear-diary/src/config/firebase.ts

```diff
@@ -3,18 +3,18 @@ import { initializeApp } from 'firebase/app'
 import { collection, getFirestore } from 'firebase/firestore'
 
 const firebaseConfig = {
-    apiKey: "AIzaSyCGUeJ-uPiiDpRXdMhr1bgwSX-Mqj9xPnc",
-    authDomain: "deardiary-2c465.firebaseapp.com",
-    projectId: "deardiary-2c465",
-    storageBucket: "deardiary-2c465.appspot.com",
-    messagingSenderId: "464384039498",
-    appId: "1:464384039498:web:52bee90de50170806a64b3",
-    measurementId: "G-C0SM8PG85Z"
-  }
+  apiKey: "AIzaSyCGUeJ-uPiiDpRXdMhr1bgwSX-Mqj9xPnc",
```

Comment:
don't store secret keys and such things directly in the code use .env file to store those things